CRM/Contact/BAO/Relationship.php

```diff
@@ -2177,6 +2187,59 @@ public static function getContactRelationshipSelector(&$params) {
     return $relationshipsDT;
   }
 
+  public static function getColumnHeaders() {
+    if (!isset(self::$_columnHeaders)) {
```

Comment:
this is dumb - someone once thought it was clever to wrap a !isset around a static array - it's not funny and it's not clever - let's not copy & paste this pattern.  Also comment block pls

Reply:
@eileenmcnaughton I'm confused by this comment - it's done nearly everywhere in the code?

Reply:
@eileenmcnaughton Waiting for you to clarify this for me :-)

Reply:
So it's not not dumb just because we do it - what does the static achieve here?

However it's non blocking

Reply:
I've removed the static